tasks/docker-compose.yaml

```diff
@@ -13,7 +13,13 @@ services:
       - high
       - default
       - low
-
+    # deploy:
```

Comment:
Not needed in rqworker right now, but we've dabbled in using gpu tasks no the rqworker before, so I'll leave this commented in.

Reply:
This will also error when trying to access a gpu on some `macos`, so isn't completely harmless to leave in uncommented. It should be useful when deploying to linux prod instances and wanting to speed up some processing by accessing the host GPU from docker.